modules/ossm-installing-primary-remote-multi-network-mesh.adoc

```diff
@@ -0,0 +1,171 @@
+// This procedure is used in the following assembly:
+// * install/ossm-multi-cluster-topologies.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="ossm-installing-primary-remote-multi-network-mesh_{context}"]
+= Installing a primary-remote multi-network mesh 
+
+Install {istio} in a primary-remote multi-network topology on two {ocp-product-title} clusters. 
+
+[NOTE]
+====
+In this procedure, `CLUSTER1` is the East cluster and `CLUSTER2` is the West cluster. The East cluster is the primary cluster and the West cluster is the remote cluster.
+====
+
+You can adapt these instructions for a mesh spanning more than two clusters.
+
+.Prerequisites
+
+* You have installed the {SMProduct} Operator on all of the clusters that comprise the mesh.
```

Comment:
I think we will have to use `OpenShift Service Mesh 3` to differentiate from 2.x operator which has a name `OpenShift Service Mesh`

Reply:
done